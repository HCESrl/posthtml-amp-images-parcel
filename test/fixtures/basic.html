<amp-img alt="Alt" src="svg/logo.svg" width="80" height="80">
  <noscript>
    <img src="/logo.ff614f16.svg" width="80" height="80" alt="Alt">
  </noscript>
</amp-img>
<amp-img alt="Alt" src="" width="80" height="80">
  <noscript>
    <img src="/logo.ff614235.svg" width="80" height="80" alt="Alt">
  </noscript>
</amp-img>
