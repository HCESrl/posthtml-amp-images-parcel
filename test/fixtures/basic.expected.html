<amp-img alt="Alt" src="/logo.ff614f16.svg" width="80" height="80">
  <noscript>
    <img src="/logo.ff614f16.svg" width="80" height="80" alt="Alt">
  </noscript>
</amp-img>
<amp-img alt="Alt" src="/logo.ff614235.svg" width="80" height="80">
  <noscript>
    <img src="/logo.ff614235.svg" width="80" height="80" alt="Alt">
  </noscript>
</amp-img>
